lib/driverProviders/driverProvider.js

```diff
@@ -32,10 +33,14 @@ DriverProvider.prototype.getExistingDrivers = function() {
  * @return webdriver instance
  */
 DriverProvider.prototype.getNewDriver = function() {
-  var newDriver = new webdriver.Builder().
-      usingServer(this.config_.seleniumAddress).
-      withCapabilities(this.config_.capabilities).
-      build();
+  var builder = new webdriver.Builder().
+    usingServer(this.config_.seleniumAddress).
+    withCapabilities(this.config_.capabilities);
+  if (!_.isUndefined(this.config_.environmentOverrides)
+      && this.config_.environmentOverrides === false) {
```

Comment:
Oh I see.  Isn't the `!_.isUndefined(this.config_.environmentOverrides)` check redundant then?  If `this.config_.environmentOverrides === false` then it certainly won't be `undefined`.